app/controllers/reviews_controller.rb

```diff
@@ -0,0 +1,52 @@
+class ReviewsController < ApplicationController
+  def show
+    @review = Review.find(params[:id])
+  end
+
+  def new
+    @release = Release.find(params[:release_id])
+    @review = Review.new
+  end
+
+  def create
+    @release = Release.find(params[:release_id])
+    if current_user
+      @review = Review.new(review_params)
+      @review.user = current_user
+      @review.release = @release
+      @review.save
+      redirect_to release_path(@release)
+    else
+      flash[:notice] = "You must be logged in to do that"
+      redirect_to new_user_session_path
+    end
```

Comment:
Please create a unhappy path, for when the user is present but the review does not get saved correctly